[FEAT] metodologias 1ª parte

daves22 opened the pull request.

daves22 committed:
feat: metodologias 1ª parte

MauricioMachadoFF commented on docs/planejamento/metodologias.md:
```diff
@@ -0,0 +1,43 @@
+
+Nossa equipe, de forma dinâmica e constante, busca melhorar e otimizar o projeto, deixando-o mais claro, com menos possibilidade de erros e de duplicação de processos, a partir de revisões e reuniões.
+
+### 2.2 Integração contínua
+
+Nossa equipe está sempre a postos para revisar o quanto antes uma nova tarefa publicada por um dos membros ao longo do desenvolvimento do projeto, para que dessa forma tenhamos um desenvolvimento mais fluido e sem interrupções indesejadas para um processo pleno. Evitando assim, problemas futuros e com a revisão contínua, corrigindo-os quando localizados.
```
Integração contínua além de revisões e alto volume de commits, tem como suporte uma estrutura de pipeline. Não temos isso.

MauricioMachadoFF commented on docs/planejamento/metodologias.md:
```diff
@@ -0,0 +1,43 @@
+
+## 3.1 Encontros semanais
+
+Além de todos os meios de comunicação por mensagem rápidas e tarefas do git, nós nos reunimos toda semana às terças-feira as 20 horas da noite na plataforma "Teams", para alinharmos nossas questões acerca do projeto e também colocarmos nossas ideias em sincronia com os demais membros da equipe. Ademais, organizamos nosso cronograma e planejamos as tarefas da próxima semana, e se de alguma forma h ...
```
```suggestion
Além de todos os meios de comunicação por mensagem rápidas e tarefas do git, nós nos reunimos toda semana nas terças-feira às 20 horas da noite na plataforma "Teams", para alinharmos nossas questões acerca do projeto e também colocarmos nossas ideias em sincronia com os demais membros da equipe. Ademais, organizamos nosso cronograma e planejamos as tarefas da próxima semana, e se de alguma forma haver alguma tarefa atrasada, colocamos como preferência e como equipe, ajudamos a terminá-la da maneira mais breve possível.
```

MauricioMachadoFF commented on docs/planejamento/metodologias.md:
```diff
@@ -0,0 +1,43 @@
+# <center>Metodologias
+
+### Histórico das versões
+|    Data     | Versão | Descrição            | Autor(es)     |
+|  :-------:  | :----: | :------------------: | :-----------: |
+|  18.11.2022 |  0.1   | Criação do documento | Davi Silva    |
```
```suggestion
|    Data     | Versão | Descrição            | Autor(es)     | Revisor(es)|
|  :-------:  | :----: | :------------------: | :-----------: | :-: |
|  18.11.2022 |  0.1   | Criação do documento | Davi Silva    | Mauricio Machado |
```

MauricioMachadoFF reviewed (commented):
Pq versão 1? Você já pensa em incrementar algo? Ou por enquanto será isso?
Por fim, onde está a referência a SCRUM? Usamos histórias de usuário, temos um backlog definido e sprints a serem entregues.

daves22 committed:
Mudanças requeridas e segunda parte

Alterei algumas coisas a partir das observações e adicionei a segunda parte da tarefa.

Co-Authored-By: Nicolas <nicolas.souza@aluno.unb.br>

nszchagas committed:
Solving merge conflicts
